tests/lib/rules/name-property-casing.js

```diff
@@ -83,8 +93,12 @@ ruleTester.run('name-property-casing', rule, {
           name: 'foo  bar'
         }
       `,
-      options: ['PascalCase'],
```

Comment:
yes, `PascalCase` is set by default and we was missing test case without options set